mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -157,6 +157,11 @@ def test_events():
     assert_raises(TypeError, read_raw_brainvision, vhdr_path, eog=eog,
                   preload=True, response_trig_shift=np.nan)
 
+    # Test that both response_trig_shit and event_id can be set
+    read_raw_brainvision(vhdr_path, eog=eog, preload=False,
+                         response_trig_shift=100,
+                         event_id={'Sync On': 5})
```

Comment:
Does this emit a warning? If so, please catch it with `warnings.catch_warnings(record=True)`. If it doesn't, it means there is some gap in your coverage. But maybe you can't get to that warning line because we don't have an appropriate file?